README.md

```diff
@@ -1,58 +1,66 @@
-SDO ANALYSIS
-============
+# SDO ANALYSIS
 
-This part of YangCatalog.org server fetches all YANG modules from different repositories and also scans all IETF RFCs and drafts for YANG modules.
+## Overview
+This container's main purpose is to collect the outputs of several open source and proprietary validators run on YANG modules: [pyang](https://github.com/mbj4668/pyang), [yanglint](https://github.com/CESNET/libyang), [yumadump-pro](https://www.yumaworks.com/tools/yang-compiler/), and [confdc](https://www.tail-f.com/management-agent/). This data is then displayed on YANG Catalog's [statistics page](https://yangcatalog.org/private-page).
 
-Once all those modules are retrieved they are validated by several tools.
+YANG modules are retrieved from several sources:
+- [YangModels/yang](https://github.com/YangModels/yang)
+- [openconfig/public](https://github.com/openconfig/public)
+- [sysrepo/yang](https://github.com/sysrepo/yang)
+- [onf/Snowmass-ONFOpenTransport](https://github.com/OpenNetworkingFoundation/Snowmass-ONFOpenTransport)
+- [openroadm/OpenROADM_MSA_Public](https://github.com/OpenROADM/OpenROADM_MSA_Public)
+- [mef/YANG-public](https://github.com/MEF-GIT/YANG-public)
+- automatically extracted from IETF Internet Drafts and RFCs
+
+When extracting data from IETF documents, the document's metadata is added to the extracted module's properties.
+
+-----------------
 
 Most of the code is in the `bin/` directory and uses `bash` and Python3 scripts.
 
 Some configuration files are in the `conf/` directory including `paths.sh` which defines all paths used by the scripts. This script merely reads the global configuration file `/etc/yangcatalog/yangcatalog.conf` and creates the required environement variables required by the shell scripts.
 
 *Some pre-requisistes are defined int he README.md in bin directory*
 
-Overall data flow
------------------
+## Overall data flow
```

Comment:
This section needs to be updated - we currently only use a `cronjob` that combines the two previous ones but uses hashing for skipping files compilation every day.
Also we use second cronjob [cronjob-drafts](https://github.com/YangCatalog/sdo_analysis/blob/master/bin/cronjobs/cronjob-drafts) which once a week checks whether modules from all (even archived) drafts are populated.